code/__HELPERS/mobs.dm

```diff
@@ -238,6 +238,32 @@ proc/age2agedescription(age)
 		if(70 to INFINITY)	return "elderly"
 		else				return "unknown"
 
+/proc/set_criminal_status(mob/living/user, datum/data/record/target_records , criminal_status, comment, user_rank, authcard_access = list())

```

Comment:
You can add the `list/` part back, I meant the `var/` part of `var/list`... should have specified 😛

Reply:
Aaaah. Thought since it was defined by the = list()